src/assets/js/partials/product-options.js

```diff
@@ -2,8 +2,29 @@ import Flatpickr from 'flatpickr';
 
 export default class ProductOptions {
     constructor() {
+        var days={};
+        let dateTime = app.element('.date-time-element');
```

Comment:
what if there are more than one date-time-element, so try to make it something like this:

```
app.all('.date-time-element',element=>{
        let days= JSON.parse(element.dataset.orderTimes||'{}');

         Flatpickr(element {
            enableTime: true,
            dateFormat: "Y-m-d H:i",
            minDate:element.dataset.fromDateTime,
            maxDate:element.dataset.toDateTime,
            enable: [
		(date) =>  !!days[date.toLocaleString('en-us', {weekday: 'long'}).toLowerCase()]
            ]
        });
})
```